spec/features/tour_list/recordings/index_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'tour list recordings', :vcr, type: :feature do
+
+  let(:user) { create(:user) }
+  let(:landmark) { create(:landmark) }
+
+  context 'as a visitor' do
+    scenario 'unable to see tour list landmarks recordings' do
+      visit tour_list_landmark_recordings_path(landmark)
+
+      expect(page.status_code).to eq(404)
+    end
+  end
+
+  context 'as a user' do
+    before :each do
+      recording = create(:recording, landmark: landmark)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+      visit landmark_path(landmark)
+      click_button 'Add to tour list'
+
+      visit tour_list_path
+    end
+
+    scenario 'can select existing recording' do
+      expect(page).to have_button('Choose Existing Recording')
+
+      within("#landmark-#{landmark.id}") do
+        click_on 'Choose Existing Recording'
+      end
+
+      expect(current_path).to eq(tour_list_landmark_recordings_path(landmark))
+
+      within(first('.recording-list')) do
+        click_on 'Pick Recording'
+      end
+
+      expect(current_path).to eq(tour_list_path)
+      expect(page).to have_content('The recording has been saved.')
+    end
+  end
+
+  context 'as an admin'
+
+  context 'edge case' do
+    scenario 'user can delete recording that was selected' do
```

Comment:
This is a different controller action, not an edge case. Should move into a separate spec file.

Reply:
I disagree. This test refers to a user deleting a recording that was selected during the create a tour functionality. You may have this confused with another CRUD action. This action does not delete or alter anything in the database.